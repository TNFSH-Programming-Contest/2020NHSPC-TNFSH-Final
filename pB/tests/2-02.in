85
-88.956 453.132
-66.516 427.102
-51.0 411.0
-8.173 414.536
6.188 374.145
-34.203 362.476
-75.492 405.56
-168.841 359.784
-123.064 296.953
-87.161 307.724
-33.306 329.266
11.574 269.127
-15.354 220.658
-64.721 252.0
-125.757 239.507
-76.39 125.514
-42.281 121.923
-53.95 208.989
17.857 163.212
17.857 112.05
-10.866 137.182
-19.842 86.02
25.935 64.478
18.754 16.008
-47.667 51.912
-39.589 5.237
-86.263 8.828
-79.082 57.297
-149.992 144.363
-161.66 14.213
-189.485 32.165
-210.0 11.52
-283.732 -0.148
-251.419 28.575
-206.54 64.478
-232.569 75.249
-291.81 38.448
-352.846 35.755
-357.334 111.152
-333.099 118.333
-330.406 71.659
-289.117 71.659
-289.117 103.074
-178.714 97.689
-176.022 181.164
-125.757 187.447
-132.938 202.706
-173.329 207.194
-240.648 262.844
-266.678 333.754
-301.684 352.603
-286.425 280.796
-350.153 286.182
-330.406 247.585
-279.244 207.194
-283.732 133.592
-385.159 131.797
-383.364 15.111
-426.448 11.52
-465.044 68.068
-444.4 228.736
-421.96 162.315
-318.738 166.803
-325.021 207.194
-396.828 200.911
-430.936 476.47
-391.442 476.47
-392.34 315.802
-318.738 315.802
-338.485 384.916
-331.304 480.958
-239.75 506.09
-179.612 503.397
-185.895 471.084
-202.949 498.012
-253.214 471.084
-265.78 387.609
-230.774 320.29
-190.383 258.356
-179.612 293.362
-219.106 360.681
-219.106 470.186
-167.046 425.307
-143.709 474.674
-116.781 410.946
17.857 112.05
