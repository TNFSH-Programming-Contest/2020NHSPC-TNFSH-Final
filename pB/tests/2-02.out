1049973.965